commons/src/main/java/fr/free/nrw/commons/MediaWikiImageView.java

```diff
@@ -151,7 +151,7 @@ private void loadImageIfNecessary(final boolean isInLayoutPass, final boolean tr
         // from the network.
         ImageContainer newContainer = mImageLoader.get(mUrl,
                 new ImageListener() {
-                    @Override
```

Comment:
It does compile, but it shows IDE errors (the squiggly red lines etc). Should I ignore those and put all the @Override's back in then?